stable/goldpinger/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+name: goldpinger
+version: 1.0.0
```

Comment:
I'd hesitate to set this to 1.0.0 right away.

Reply:
Thank you for the review. Which version should I set?

Usually, reviewers ask me to set version 1.0.0.

Reply:
if we're calling it stable ... then 1.0.0 is good!